### DISCUSSION TITLE
Add separate keycounter to alternate buttons in standard

### DISCUSSION THREAD

**Discussion by tadatomix** [ppy/osu#28308] [Ideas] [answered]
Not exactly sure how you all feel or think about this, but for me it's strange to not count alternate buttons, but count smoke button, which never was counted in stable. Is there actually a reason for doing this?

**Comment by Joehuu** [comment, marked as answer]
This was asked before in https://github.com/ppy/osu/discussions/13043.

**Reply by tadatomix** [reply]
Oh, but it's still unanswered...
